.github/ISSUE_TEMPLATE/feature-request_en.yml

```diff
@@ -0,0 +1,29 @@
+name: ✨ Feature Request
+description: Suggest an improvement or added feature
+title: "[✨ Request]: "
+labels: [ "request" ]
+
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        Before creating a feature request, make sure you are using the latest version of [LibertyBans](https://jenkins.hahota.net/job/LibertyBans/) and that it isn't already implemented.
+ 
+  - type: checkboxes
+    id: preliminary
+    attributes:
+      label: I have confirmed that ...
+      options:
+        - label: There is no duplicate Feature Request.
+          required: true
+        - label: This request should *NOT* be out of scope for LibertyBans.
```

Comment:
```suggestion
```
This is a matter of judgment which can't be easily reduced to a single check-box. When users have a feature they honestly believe would better LibertyBans, I'm happy to entertain it and discuss whether it is inside or outside of scope.